src/autopas/pairwiseFunctors/LJFunctorAVX.h

```diff
@@ -68,11 +69,13 @@ class LJFunctorAVX : public Functor<Particle, ParticleCell, typename Particle::S
         _highCorner(highCorner),
         _postProcessed{false} {
     if (calculateGlobals or duplicatedCalculation) {
-      utils::ExceptionHandler::exception("Calculation of global values and duplicated calculations not supported.");
+      /*
+       *utils::ExceptionHandler::exception("Calculation of global values and duplicated calculations not supported.");
```

Comment:
If the exception is no longer needed remove also the `if`

Reply:
Removed exception and `if`

Added exception from LJFunctor.h to check if lowCorner and highCorner are properly specified.